exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandlerProtoTest.java

```diff
@@ -336,9 +349,9 @@ private void generateSpan_WithResourceLabels(Map<String, AttributeValue> resourc
             CHILD_SPAN_COUNT,
             status,
             endTimestamp);
-    Span span = handler.generateSpan(spanData, resourceLabels);
+    Span span = handler.generateSpan(spanData, getResourceLabels(resource));
```

Comment:
I'm not sure if this would work, but I was wondering if it would be better to combine the original versions of this helper method and the three tests above into one test for `generateSpan` (only testing one of the MonitoredResource types), and write separate tests for `getResourceLabels` that test all three MonitoredResource types.